src/components/navbar/Navbar.jsx

```diff
@@ -8,11 +10,19 @@ import whatsappIcon from '@/../src/../public/icons/whatsapp.svg'
 import facebookIcon from '@/../src/../public/icons/facebook.svg'
 
 const Navbar = () => {
+  const pathname = usePathname()
+  // CLOSE SIDE NAVBAR
```

Comment:
I think this comment is not necessary anymore since the function name does a good job describing its purpose :)